templates/kubernetes-cluster.template

```diff
@@ -381,6 +383,36 @@ Resources:
             # so pods can communicate
             kubectl apply -f https://${QSS3BucketName}.s3.amazonaws.com/${QSS3KeyPrefix}/scripts/${NetworkingProvider}.yaml
 
+            # Expose the api server over ELB
+            kubectl -n kube-system expose pod \
```

Comment:
Does the ELB depend on the master?  I'm not planning on adding any backends to it, I was hoping (and this is just a theory) that creating it and tagging it will be enough, and not adding anything to it.  From reading the aws cloud provider code I was under the impression that it tolerates the load balancer existing but not being configured.